### ISSUE TITLE
Obsolete asset location

### ISSUE THREAD

**Issue by Guite** [zikula/core#4028]
Checking for overrides in [the Asset class](https://github.com/zikula/core/blob/master/src/system/ThemeModule/Engine/Asset.php) needs to be updated after #4052 has been done.

Refs new directory layout in Symfony 4/5

**Comment by craigh** [comment]
I think this can be re-closed @Guite

**Comment by Guite** [comment]
Closing in favour of #4052
